### ISSUE TITLE
In the Big-Bang-Umbrella example Jira has port errors

### ISSUE THREAD

**Issue by david--wright** [defenseunicorns/zarf-package-software-factory#21]
### Steps to Reproduce:

Package and deploy big bang umbrella example

### Actual Result:
Jira displays port and host errors

### Expected Result:
Jira should be configured for the in use port

### Attempted fix:

1. In the Big-Bang-Umbrella example edit manifests/atlassian-manifests.yaml add the following under spec:ValuesContent:
    jira:
      additionalEnvironmentVariables:
        - name: ATL_PROXY_NAME
          value: "jira.bigbang.dev"
        - name: ATL_PROXY_PORT
          value: "9443"
        - name: atl_proxy_name
          value: "jira.bigbang.dev"
        - name: atl_proxy_port
          value: "9443"
2. Package and Deploy the example

### Fix Actual Result:
Environmental variables are set in the container as expected but the Jira instance does not use them. Jira displays:
```
Dashboard Diagnostics: Mismatched URL Port
Jira is reporting that it is running on the port '443', which does not match the hostname used to run these diagnostics, '9443'. This is known to cause Jira to construct URLs using the incorrect port, which will result in errors in the dashboard, among other issues.

The most common cause of this is the use of a reverse-proxy HTTP server (often Apache or IIS) in front of the application server running Jira. While this configuration is supported, some additional setup might be necessary in order to ensure that Jira detects the correct port.
```

### Fix Expected Result:
Jira should use the environmental variables to set the proxy name and port resolving this error

### Things of note:
The documentation says to use ATL_PROXY_PORT to set the port. The code in the server.xml.j2 file looks at atl_proxy_port.

The container has both a server.xml file in the application folder(/opt/atlassian/jira/conf/server.xml) and a server.xml.j2 file in /opt/atlassian/etc. The server.xml.j2 refers to atl_proxy_port to pull the proxy port. server.xml has 443 set as the proxy port.

The easiest solution would be to run Big-Bang on port 443 as that would prevent the need for the env variables to be set. It sounds like https://github.com/defenseunicorns/zarf/issues/121 might resolve this?

**Comment by RothAndrew** [comment]
Blocked by #24

**Comment by RothAndrew** [comment]
Resolved by #30. Closing.
